packages/shared/src/modules/users/libs/enums/user-validation-message.enum.ts

```diff
@@ -1,6 +1,10 @@
+import { UserValidationRule } from "./user-validation-rule.enum.js";
+
 const UserValidationMessage = {
 	EMAIL_REQUIRE: "Email is required",
 	EMAIL_WRONG: "Email is wrong",
+	PASSWORD_MAX_LENGTH: `Password must be at most ${String(UserValidationRule.PASSWORD_MAX_LENGTH)} characters long`,
```

Comment:
Do we need String() conversion here? I believe it will be converted automatically

Reply:
I'm encountering this eslint error if I don't convert it.

error  Invalid type "30" of template literal expression
error  Invalid type "3" of template literal expression

Reply:
![Screenshot 2024-08-19 143215](https://github.com/user-attachments/assets/c290d944-5993-4578-96e8-3f7dd10b23f2)

Reply:
Keep String in that case